scripts/main.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const url = 'https://jsonplaceholder.typicode.com/';
+const tableBody = document.getElementById('table_body');
+
+function getInfo(URL) {
+  return fetch(URL)
+                  .then(responce => responce.json());
+}
+
+async function createTable() {
+  const [toDos, users] = await Promise.all([getInfo(`${url}todos`),
```

Comment:
спробуй записати якось так
```suggestion
  const [toDos, users] = await Promise.all(
    [getInfo(`${url}todos`), getInfo(`${url}users`)]
  );
```